src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlTypes/SqlDateTime2.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Data.SqlTypes;
+using System.Runtime.InteropServices;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+//using Microsoft.Data.Common;
```

Comment:
Remove on used libraries.

Reply:
I would never leave it in, but VS can build with that using and using the ADP class - when I try to build outside of VS using the `msbuild /T:Release` command it fails with the error that it cannot find that namespace - so when I have had help figuring out why I cannot build - I will include it again and also the ADP class

Reply:
my build commands are:
```
msbuild /t:clean
msbuild /p:Configuration=Release
msbuild /t:BuildTestsNetCore /p:TargetNetCoreVersion=netcoreapp3.1 /p:Configuration=Release
```
without some of those args it can get a bit confused about what it's building and why.

Reply:
No dice - I still get:

````
"D:\git\SqlClient\build.proj" (default target) (1) ->
"D:\git\SqlClient\src\Microsoft.Data.SqlClient\netcore\src\Microsoft.Data.SqlClient.csproj" (default target) (4:33) ->
"D:\git\SqlClient\src\Microsoft.Data.SqlClient\netcore\src\Microsoft.Data.SqlClient.csproj" (Build target) (4:36) ->
  D:\git\SqlClient\src\Microsoft.Data.SqlClient\src\Microsoft\Data\SqlTypes\SqlDateTime2.cs(7,22): error CS0234: The type or namespace name 'Common' does not exist in the namespace 'Microsoft.Data' (are you missing an assembly reference?) [D:\git\SqlClient\src\Microsoft.Data.SqlClient\netcore\src\Microsoft.Data.SqlClient.csproj]
````

If you have a local copy, can you try to mimic what I have done, i.e. in a class that lives in the `src\Microsoft.Data.SqlClient\src` common area include 
using Microsoft.Data.Common;
and reference the ADP class - just to see if you can reproduce it.

Reply:
I have even tried now mimicking the azure build command line:
```
msbuild build.proj /nologo /nr:false /t:BuildAllConfigurations /p:GenerateNuget=false /p:configuration="Release"
```
And that does not work either - it seems like visual studio does some "magic" with the common files linked in.

I will investigate further by adding diagnostics logging to msbuild to see why it seems like its not picking up the namespace/ADP class

Reply:
I think this might the the reason why its not working for me:

in the Microsoft.Data.SqlClient.csproj for netcore:

```
<ItemGroup Condition="'$(OSGroup)' != 'AnyOS'">
```

Where the file for the ADP class is defined under - so I guess I would need to move SqlDateTime2.cs up in the same ItemGroup.